test/metabase/task/cleanup_temorary_computation_job_results_test.clj

```diff
@@ -0,0 +1,27 @@
+(ns metabase.task.cleanup-temorary-computation-job-results-test
+  (:require [clj-time
+             [coerce :as t.coerce]
+             [core :as t]]
+            [expectations :refer :all]
+            [metabase.models
+             [computation-job :refer [ComputationJob]]
+             [computation-job-result :refer [ComputationJobResult]]]
+            [metabase.task.cleanup-temporary-computation-job-results :as task]
+            [toucan.db :as db]
+            [toucan.util.test :as tt]))
+
+(tt/expect-with-temp
+ [ComputationJob       [{job-id :id} {:type   :simple-job
+                                      :status :done}]
+  ComputationJobResult [{result-id :id} {:job_id     job-id
+                                         :permanence :temporary
+                                         :payload    1}]]
+  true
```

Comment:
You shouldn't use `expect-with-temp` if the `expected` form doesn't need it. Just do

```clojure
(expect
  (tt/with-temp* ...))
```

(`expect` with one are is the same as `(expect true ...)`